### PR TITLE
working-groups: add pitr working group

### PR DESCRIPTION
**Pull request by XuHuaiyu**

### CODE REVIEW

File: working-groups/wg-pitr.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+## Organizers
+
+* WangXiangUSTC (**[@wangxiangustc](https://github.com/WangXiangUSTC)**), PingCAP
+* HuaiyuXu (**[@huaiyuxu](https://github.com/XuHuaiyu)**), PingCAP
+* AilinKid (**[@AilinKid](https://github.com/AilinKid)**), PingCAP
+* tsthght(**[@tsthght](https://github.com/tsthght)**), MeiTuan
+
+## Contact
+- Slack: [#pitr-wg](https://tidbcommunity.slack.com/archives/CRH5594F8)
```
Comment (winkyao) [inline working-groups/wg-pitr.md]: we can name the channel as `wg-pitr`.

**Comment by XuHuaiyu** [comment]
PTAL @winkyao

**Comment by winkyao** [comment]
@zz-jason @dcalvin @gingerkidney PTAL

File: working-groups/wg-pitr.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# PITR Working Group
+
+PITR is an ecosystem tool for TiDB Binlog. By preprocessing the incremental backup file of TiDB, PITR merged the changes of the same line of data to produce a new, lighter incremental backup file, which greatly reduced the Time of incremental backup Recovery and realized fast-pitr (Fast Point in Time Recovery).
```
Comment (zz-jason) [inline working-groups/wg-pitr.md]: what does `PITR` mean?

File: working-groups/wg-pitr.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Stakeholder SIGs
```
Comment (zz-jason) [inline working-groups/wg-pitr.md]: we can remove this section if it's not determinate

File: working-groups/wg-pitr.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+To be discussed.
+
+## Meetings
+
+To be discussed.
```
Comment (zz-jason) [inline working-groups/wg-pitr.md]: please add public meeting notes and meeting zoom for this working group.

File: working-groups/wg-pitr.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+## Contact
+- Slack: [#wg-pitr](https://tidbcommunity.slack.com/archives/CRH5594F8)
+- [Open Community Issues/PRs](https://github.com/pingcap/community/issues/126)
+
+## Published Documents
+
+https://github.com/lvleiice/Better-PITR
+
+### RFC/Proposal
```
Comment (zz-jason) [inline working-groups/wg-pitr.md]: I think it's included in the `Published Documents`, maybe this section can be removed

File: working-groups/wg-pitr.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+* AilinKid (**[@AilinKid](https://github.com/AilinKid)**), PingCAP
+* tsthght(**[@tsthght](https://github.com/tsthght)**), MeiTuan
+
+## Contact
+- Slack: [#wg-pitr](https://tidbcommunity.slack.com/archives/CRH5594F8)
+- [Open Community Issues/PRs](https://github.com/pingcap/community/issues/126)
+
+## Published Documents
+
+https://github.com/lvleiice/Better-PITR
```
Comment (zz-jason) [inline working-groups/wg-pitr.md]:
```
* Design Doc: [中文 (Chinese)](https://github.com/lvleiice/Better-PITR/blob/master/README.md)
```

File: working-groups/wg-pitr.md (hunk 2 above)
Comment (WangXiangUSTC) [reply working-groups/wg-pitr.md]: Point in Time Recovery

Comment (zz-jason) [reply working-groups/wg-pitr.md]: Please add this explanation to the documents.

File: working-groups/wg-pitr.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
+# PITR Working Group
+
+PITR(Point in Time Recovery) is an ecosystem tool for TiDB Binlog. By preprocessing the incremental backup file of TiDB, PITR merged the changes of the same line of data to produce a new, lighter incremental backup file, which greatly reduced the Time of incremental backup Recovery and realized fast-pitr.
```
Comment (dcalvin) [inline working-groups/wg-pitr.md]:
```suggestion
PITR (Point in Time Recovery) is an eco-system tool for TiDB Binlog. By preprocessing the incremental backup file of TiDB, PITR merges the changes of the same row of data to produce a new, lighter incremental backup file. This greatly reduces the time consumed for incremental backup recovery and achieves a fast point in time recovery.
```

Comment (dcalvin) [reply working-groups/wg-pitr.md]: > realized fast-pitr

this should be used in the descriptive sense, not the feature name.

**Pull request merged**